pyannote/database/custom.py

```diff
@@ -169,17 +169,25 @@ def subset_iter(database_name, file_lst=None, file_rttm=None,
         # initialize current file with the mandatory keys
         current_file = {'database': database_name, 'uri': uri}
 
+        # add 'annotated' when UEM file is provided
+        # defaults to empty Timeline for the same reason as above
+        if file_uem is not None:
+            current_file['annotated'] = annotated.get(uri, Timeline(uri=uri))
+
         # add 'annotation' when RTTM file is provided
         # defaults to empty Annotation because of
         # github.com/pyannote/pyannote-database/pull/13#discussion_r261564520)
         if file_rttm is not None:
             current_file['annotation'] = annotations.get(
                 uri, Annotation(uri=uri))
-
-        # add 'annotated' when UEM file is provided
-        # defaults to empty Timeline for the same reason as above
-        if file_uem is not None:
-            current_file['annotated'] = annotated.get(uri, Timeline(uri=uri))
+            if current_file['annotated']:
```

Comment:
This will fail with a `KeyError` if there is no `annotated` section in `database.yml`.

Reply:
yes I thought that current_file['annotated'] was always an empty timeline by default, I was about to fix that